test/new-e2e/tests/process/testing.go

```diff
@@ -189,22 +189,17 @@ func findProcess(
 func filterProcesses(name string, processes []*agentmodel.Process) []*agentmodel.Process {
 	var matched []*agentmodel.Process
 	for _, process := range processes {
-		if len(process.Command.Args) > 0 && process.Command.Args[0] == name {
+		if matchProcess(process, name) {
 			matched = append(matched, process)
 		}
 	}
 	return matched
 }
 
-// filterProcess returns process with the given name exists in the given list of processes
-func filterProcess(name string, processes []*agentmodel.Process) []*agentmodel.Process {
```

Comment:
Removing as this method is a duplicate of `filterProcesses` just above this.

Reply:
How did this happen 😅